deps.bzl

```diff
@@ -173,8 +173,8 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
     go_repository(
         name = "com_github_azure_go_autorest_autorest",
         importpath = "github.com/Azure/go-autorest/autorest",
-        sum = "h1:gI8ytXbxMfI+IVbI9mP2JGCTXIuhHLgRlvQ9X4PsnHE=",
-        version = "v0.11.12",
+        sum = "h1:gI8ytXbxMfI+IVbI9mP2JGCTXIuhHLgRlvQ9X4PsnHE=",  # keep
```

Comment:
My suggestion is to not fight gazelle here and let it downgrade the dependencies. This will be easier to maintain going forward. This matches the behavior of plain Go. 

For transitive dependencies, Go picks the highest compatible version amongst all the modules in the graph that depend on it. You can see the dependency graph by running `go mod graph`.

Reply:
Fixed!